src/assimp.mk

```diff
@@ -11,16 +11,15 @@ $(PKG)_DEPS     := cc boost minizip
 
 define $(PKG)_BUILD
     cd '$(BUILD_DIR)' && $(TARGET)-cmake \
-        -DASSIMP_ENABLE_BOOST_WORKAROUND=OFF \
         -DASSIMP_BUILD_ASSIMP_TOOLS=OFF \
         -DASSIMP_BUILD_SAMPLES=OFF \
         -DASSIMP_BUILD_TESTS=OFF \
         '$(SOURCE_DIR)'
     $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)'
     $(MAKE) -C '$(BUILD_DIR)' -j 1 install
 
-    '$(TARGET)-gcc' \
-        -W -Wall -Werror -ansi -pedantic \
+    '$(TARGET)-g++' \
+        -W -Wall -Werror -ansi -pedantic -std=c++11 \
         '$(TEST_FILE)' -o '$(PREFIX)/$(TARGET)/bin/test-assimp.exe' \
-        `'$(TARGET)-pkg-config' assimp minizip --cflags --libs`
+        `'$(TARGET)-pkg-config' assimp minizip --cflags --libs` -lirrxml
```

Comment:
The lib is named `libIrrXML` and fails on case-sensitive systems for all targets.

Also, the shared builds don't install a *.dll version of IrrXML.

Reply:
> zstd was recently added to mxe and can be added to DEPS and enabled
> 
> Edit: that applies to minizip

I know, I explicitly disabled it though because the build fails at linking, maybe in the future it will work properly.

Reply:
Apparently this was a bug upstream. I'm currently investigating more.

Reply:
So.. the shared version of assimp links IrrXML statically, which is why it obviously doesn't install it (also it's a third-party lib so why would they install it?) but now I'm trying to understand why the static version DOES install it.